test-scripts/local-is-deployment/DeploymentPatterns/pattern-2/deploy.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# This script depends on curl and unzip, so make sure they are already installed.
+
+echo "This is a Dummy Deployment creation scripts to deploy a standalone WSO2 Identity server"
+echo "Here the test infrastructure is your local machine."
+pwd
+#if you do not have a account, please create a account at https://wso2.com/
+WOS2_USER=yasassri@wso2.com
+WSO2_PASSWORD=xxxxxxx
```

Comment:
These are unnecessary. WSO2 products are not available over CURL. First variable name has a typo too..

Reply:
We can parse UT credentials over curl to download the product distros. I have commented that line for now, And users can uncomment it if they wish to download the pack as well.